pandera/api/pandas/types.py

```diff
@@ -81,6 +82,9 @@ def is_table(obj):
     return isinstance(obj, supported_types().table_types)
 
 
+IsTable = parametric(object, is_table)
```

Comment:
are these changes still compatible with older versions of multimethod?

Reply:
I'd added the lower bound, but now I pulled out the `parametric` dependency, so I'll unpin the requirement again.